crates/noirc_evaluator/src/ssa_refactor.rs

```diff
@@ -24,9 +24,15 @@ pub mod ssa_gen;
 /// Optimize the given program by converting it into SSA
 /// form and performing optimizations there. When finished,
 /// convert the final SSA into ACIR and return it.
-pub fn optimize_into_acir(program: Program) -> Acir {
-    ssa_gen::generate_ssa(program).inline_functions().into_acir()
+pub fn optimize_into_acir(program: Program) {
+    ssa_gen::generate_ssa(program)
+        .print("Initial SSA:")
+        .inline_functions()
+        .print("After Inlining:")
+        .unroll_loops()
+        .print("After Unrolling:");
```

Comment:
This changes `--experimental-ssa` flag to print out the IR by default. I suppose this should instead require `--show-ssa` as well

Reply:
Should be okay while we are using it as experimental me thinks